test-configs.yaml

```diff
@@ -568,6 +573,11 @@ device_types:
     mach: amlogic
     class: arm64-dtb
     boot_method: uboot
+    kernel_addr: '0x01080000'
```

Comment:
These attributes are specific to the `fastboot` method, unlike all the other attributes.  So it should be a dictionary of arbitrary options passed to the LAVA job template rather than going through the device settings.